packages/http/src/validator/index.ts

```diff
@@ -37,7 +38,14 @@ const validateInput: ValidatorFn<IHttpOperation, IHttpRequest> = ({ resource, el
 const validateOutput: ValidatorFn<IHttpOperation, IHttpResponse> = ({ resource, element }) => {
   const results: IPrismDiagnostic[] = [];
   const mediaType = caseless(element.headers || {}).get('content-type');
-  const responseSpec = resource.responses && findOperationResponse(resource.responses, element.statusCode);
+  const responseSpec = resource.responses && toUndefined(findOperationResponse(resource.responses, element.statusCode));
+
+  if (!responseSpec) {
```

Comment:
…in this specific case, we have two paths: one when the responseSpec is not defined (we add an element in the array)

Reply:
Can we avoid the word spec? It’s a description doc

Reply:
Changed to `responseDescDoc`.

@philsturgeon Why is `description doc` better than `spec`?

Reply:
I would say consistency; if we're using description Document in Stoplight generally, it makes sense to stick to the same convention everywhere; the spec — I think — refers to the specification (oas2, oas3)

Reply:
I the poll is still open my vote goes on `specification`. The world calls it `specification` and I don't think it is good to invent new names. Especially longer ones ;)